<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="send_email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_to">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>169</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email получателя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тема письма:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Header_email">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>209</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_email">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>249</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст письма:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сервер</string>
    </property>
   </widget>
   <widget class="QComboBox" name="server_selection">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Яндекс</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Google</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mail.ru</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес отправителя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_from">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль от аккаунта</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_password">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Прикрепить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attach_file">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="files_to_be_send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEmailer">
    <property name="title">
     <string>Emailer</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuEmailer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
